azure/service-bus/src/main/scala/com/commercetools/queue/azure/servicebus/ServiceBusMessageContext.scala

```diff
@@ -35,7 +36,9 @@ class ServiceBusMessageContext[F[_], T](
   override def enqueuedAt: Instant = underlying.getEnqueuedTime().toInstant()
 
   override def metadata: Map[String, String] =
-    Map.empty
+    underlying.getRawAmqpMessage.getApplicationProperties.asScala.view.collect {
+      case (k, v) if v.isInstanceOf[String] => (k, v.asInstanceOf[String])
```

Comment:
Isn't this way a bit more idiomatic?

```suggestion
      case (k, v: String) => (k, v)
```